src/main/java/org/broadinstitute/hellbender/tools/variantdb/ExtractCohortBQ.java

```diff
@@ -32,10 +47,84 @@
 
         // Add our samples to our map:
         for (final FieldValueList row : result.iterateAll()) {
-            results.add(row.get(0).getStringValue());
+            results.put((int) row.get(0).getLongValue(), row.get(1).getStringValue());
+        }
+
+        return results;
+    }
+
+    private static String getOptionalString(FieldValue v) {
+        return (v == null || v.isNull()) ? null : v.getStringValue();
+    }
+
+    public static Map<Long, ProbeInfo> getProbeIdMap(String fqProbeTableName, boolean printDebugInformation) {
+    
+        Map<Long, ProbeInfo> results = new HashMap<>();
+
+        // Get the query string:
+        final String sampleListQueryString = 
+            "SELECT probeId, Name, Chr, Position, Ref, AlleleA, AlleleB" + 
+            " FROM `" + fqProbeTableName + "`";
```

Comment:
we could add a constant in ProbeInfoSchema for the list of field to get for extract. still need to figure out a package structure that makes sense since right now that class is in ingest